src/app/modules/group/pages/current-user/current-user.component.html

```diff
@@ -1,5 +1,18 @@
 <ng-container *ngIf="currentUser$ | async as currentUser">
   <div class="bg-white">
+    <div class="modify-section">
+      <div class="flex-1" i18n>
+        The following information is what what receive from France-IOI login platform.
```

Comment:
Although that's exactly what's written in the issue, I doubt that "The following information is **what what** .." is correct :smile: 
I suppose: "The following information is what **we** receive from France-IOI login platform."

Reply:
Fixed